ext/ForwardDiffGPUArraysCoreExt.jl

```diff
@@ -0,0 +1,62 @@
+module ForwardDiffGPUArraysCoreExt
+
+using GPUArraysCore: AbstractGPUArray
+using ForwardDiff: ForwardDiff, Dual, Partials, npartials, partials
+
+function ForwardDiff.seed!(duals::AbstractGPUArray{Dual{T,V,N}}, x,
+                           seed::Partials{N,V} = zero(Partials{N,V})) where {T,V,N}
```

Comment:
It's not necessary to duplicate the default values here:
```suggestion
                           seed::Partials{N,V}) where {T,V,N}
```